list-pr-files/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "list-pr-files",
+  "version": "1.0.0",
+  "description": "List pull request files",
+  "main": "dist/index.js",
+  "repository": "git@github.com:matheuspiment/list-pr-files.git",
```

```suggestion
  "repository": "git@github.com:exercism/github-actions.git",
```

We should add directory, but I'm on my phone :p